control-plane/subcommand/gateway-cleanup/command.go

```diff
@@ -19,29 +21,44 @@ import (
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	gwv1beta1 "sigs.k8s.io/gateway-api/apis/v1beta1"
+	k8syaml "sigs.k8s.io/yaml"
 
+	"github.com/hashicorp/consul-k8s/control-plane/api/mesh/v2beta1"
 	"github.com/hashicorp/consul-k8s/control-plane/api/v1alpha1"
 	"github.com/hashicorp/consul-k8s/control-plane/subcommand"
 	"github.com/hashicorp/consul-k8s/control-plane/subcommand/flags"
 )
 
+const (
+	gatewayConfigFilename  = "/consul/config/config.yaml"
+	resourceConfigFilename = "/consul/config/resources.json"
```

Comment:
should these live in another package? we use them in the resources job so I'm wondering if they belong in the `gateways` package as constants so we're not defining them twice across this and the resources job

Reply:
I'd be in favor of pulling them into a shared package.